docs-new/docs/contribute/general/contrib-guidelines-gen.md

```diff
@@ -43,3 +43,12 @@ please create an issue on the GitHub repository.
 * If you want to do the work on an issue,
   include that information in your description of the issue
   or in a comment to the issue.
+
+## Proposing New Work
+
+* Create and Refine a Ticket
+  * When proposing new work, start by creating an issue or ticket in the project's issue tracker.
```

Comment:
Should this specify the github issue tracker?  Maybe even link to it?

Reply:
Sure will add the issue tracker link over here